lab_1/main.py

```diff
@@ -26,7 +43,16 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
+    tokens2 = []
+    if type(tokens) == list:
+        if type(stop_words) == list:
+            for i in stop_words:
+                if i in tokens:
+                    tokens.remove(i)
```

Comment:
try this:
stop_words = ['the', 'is']
tokens = ['the', 'is', 'a', 'the']

The expected result is ['a']